nose_launchable/plugin.py

```diff
@@ -52,30 +49,42 @@ def configure(self, options, conf):
         super(Launchable, self).configure(options, conf)
 
         self.subset_enabled = options.subset_enabled or False
+        self.record_only_enabled = options.record_only_enabled or False
```

Comment:
AIU, those are two primary behavior modes so I collocated them to show the symmetry